app/marketing/views.py

```diff
@@ -268,6 +268,30 @@ def email_settings(request, key):
     email = ''
     level = ''
     msg = ''
+    email_types = dict([('welcome_mail', 'Welcome Email'),('roundup', 'Round Up'), ('new_bounty_notifications', 'New Bounty Notifications Emails'), ('important_product_updates', 'Product Updates Emails'), ('tip', 'Tip Emails'), ('faucet', 'Faucet Notification Emails'), ('bounty', 'Bounty Notification Emails'), ('bounty_match', 'Bounty Match Emails'), ('bounty_feedback', 'Bounty Feedback Emails'), ('bounty_expiration', 'Bounty Expiration Warning Emails'), ('featured_funded_bounty', 'Featured Funded Bounty Emails')])
+    if request.GET.get('type') in email_types:
+        email = request.user.profile.email
+        dictionary = {}
```

Comment:
Since this variable is only used within the `if es:` scope, lets not declare it before the scope begins. Also, lets use a better name than `dictionary`.

Reply:
+1